### ISSUE TITLE
[Bug] Error E1101 when accessing `__annotations__` on `TypedDict` classes

### ISSUE THREAD

**Issue by Vulcagon** [microsoft/vscode-pylint#539]
When accessing the `__annotations__` attribute on a `TypedDict` Pyint gives the error

> Instance of `TypedDict` has no `__annotations__` member
> [E1101:no-member](https://pylint.readthedocs.io/en/latest/user_guide/messages/error/no-member.html)

This only happens when defining a TypedDict like `MyTypedDict = TypedDict(...)`

```python
from typing import TypedDict


class MyTypedDict(TypedDict):
    a: int
    b: str


OtherTypedDict = TypedDict('OtherTypedDict', {'a': int, 'b': str})


print(MyTypedDict.__annotations__)  # No Error
print(OtherTypedDict.__annotations__)  # Error
```

I'm using Pylint as VS Code extension

Python version: 3.12.2
VS Code version: 1.87.2 (Windows 11)
Python extension version: 2024.2.1
Pylint extension version: 2023.10.1

**Comment by karthiknadig** [comment]
@Vulcagon This should be filed on the `pylint` repo: https://github.com/pylint-dev/pylint/issues

I can repo this from command line. This extension is just a wrapper around pylint making it easier to run and see the results from pylint.
![image](https://github.com/microsoft/vscode-pylint/assets/3840081/756fe58b-1198-447a-9c52-b00a93aa1458)

**Comment by Vulcagon** [comment]
@karthiknadig Oh thank you. I found out that this issue has already been reported [here](https://github.com/pylint-dev/pylint/issues/7126)

Quick question: Where do I report issues when you're code is colorized incorrectly? On the [vscode-python](https://github.com/microsoft/vscode-python/issues) repo?

eg
```python
print(f"[{
    my_var
}]")
 ^ ^
```

**Comment by karthiknadig** [comment]
Try https://github.com/microsoft/pylance-release/issues
